Source/JavaScriptCore/runtime/Error.cpp

```diff
@@ -255,6 +255,34 @@ JSObject* addErrorInfo(VM& vm, JSObject* error, int line, const SourceCode& sour
     return error;
 }
 
+JSObject* createTypeErrorCopy(JSGlobalObject* globalObject, JSValue error)
+{
+    VM& vm = globalObject->vm();
+    auto scope = DECLARE_CATCH_SCOPE(vm);
+    String errorString = "Error encountered during evaluation"_s;
+
+    if (error.isPrimitive())
+        errorString = error.toWTFString(globalObject);
```

Comment:
We need exception check here to suppress warnings.

Reply:
Would it be terrible to add a version of toWTFString that doesn't throw or invoke user code?

Reply:
Regardless, done